network-store-server/src/main/java/com/powsybl/network/store/server/NetworkStoreRepository.java

```diff
@@ -1168,10 +1178,10 @@ public void updateVscConverterStationsSv(UUID networkUuid, List<Resource<Injecti
         updateInjectionsSv(networkUuid, resources, VSC_CONVERTER_STATION_TABLE);
     }
 
-    public void deleteVscConverterStation(UUID networkUuid, int variantNum, String vscConverterStationId) {
-        deleteIdentifiable(networkUuid, variantNum, vscConverterStationId, VSC_CONVERTER_STATION_TABLE);
-        deleteReactiveCapabilityCurvePoints(networkUuid, variantNum, vscConverterStationId);
-        deleteRegulatingPoints(networkUuid, variantNum, Collections.singletonList(vscConverterStationId), ResourceType.VSC_CONVERTER_STATION);
+    public void deleteVscConverterStations(UUID networkUuid, int variantNum, List<String> vscConverterStationIds) {
+        deleteIdentifiables(networkUuid, variantNum, vscConverterStationIds, VSC_CONVERTER_STATION_TABLE);
+        deleteReactiveCapabilityCurvePoints(networkUuid, variantNum, vscConverterStationIds);
```

Comment:
remove the methods where there was one id as argument as they're not used anymore